@navikt/core/react/src/inline-message/root/InlineMessage.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { forwardRef } from "react";
+import { useRenameCSS, useThemeInternal } from "../../theme/Theme";
+import { BodyShort } from "../../typography";
+import type { OverridableComponent } from "../../util";
+import { InlineMessageIcon } from "../icon/InlineMessageIcon";
+
+interface InlineMessageProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * InlineMessage variant.
+   */
+  variant: "info" | "success" | "warning" | "error";
+  /**
+   * InlineMessage size.
+   * @default "medium"
+   */
+  size?: "medium" | "small";
+}
+
+/**
+ * InlineMessage is used to display important messages together with other content.
```

Comment:
I think this one perhaps also could be more like the docs.

```suggestion
 * InlineMessage is used to highlight short messages next to other content.
```